src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -41,6 +46,40 @@
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+
+        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
incorrect constant field
please read message in slack or see here https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names